.gitattributes

```
@@ -1 +1,2 @@
 rust/ommx/src/ommx.v1.rs linguist-generated=true
+python/ommx/* linguist-generated=true
```

```suggestion
python/ommx/**/* linguist-generated=true
```

`*`はディレクトリ以下見ないので、そのディレクトリ以下を全部指定したいならこうです
https://www.git-scm.com/docs/gitignore#_pattern_format